mods/voidmulti.sc2mod/base.sc2data/gamedata/weapondata.xml

```diff
@@ -141,6 +153,7 @@
     <CWeaponLegacy id="NeedleSpines">
         <Period value="0.825"/>
         <MinScanRange value="5.5"/>
+        <DamagePoint value="0.14"/>
```

Comment:
Hydralisk attack's Damage point reduced from 0.15 to 0.1 (from 0.208 to 0.14 game seconds)

This is the ranged attack with the projectile